packages/sdk/src/conversationWithCloudAdapter/interface.ts

```diff
@@ -0,0 +1,524 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CloudAdapter,
+  ConversationState,
+  UserState,
+  Activity,
+  TurnContext,
+  InvokeResponse,
+  Storage,
+  SigninStateVerificationQuery,
+} from "botbuilder";
+import { TeamsBotSsoPromptTokenResponse } from "../bot/teamsBotSsoPromptTokenResponse";
+import {
+  AuthenticationConfiguration,
+  OnBehalfOfCredentialAuthConfig,
+} from "../models/configuration";
+
+/**
+ * The response of a message action, e.g., `sendMessage`, `sendAdaptiveCard`.
+ */
+export interface MessageResponse {
```

Comment:
For developers, does new interface need to be `BotBuilderCloudAdapter.xxx` as well?
Any thoughts on deprecating old interfaces or reusing some interfaces?

Reply:
Yes for the first question.
Deprecating and reusing some interfaces are good ideas. Just removed duplicated interfaces from `conversationWithCloudAdapter` and reuse interfaces from `conversation`. Old interfaces such as `NotificationOptions` and `ConversationOptions` are deprecated.
Thanks!